src/test/ui/consts/const-multi-ref.rs

```diff
@@ -2,7 +2,7 @@ const _X: i32 = {
     let mut a = 5;
     let p = &mut a;      //~ ERROR references in constants may only refer to immutable values
 
-    let reborrow = {p};  //~ ERROR references in constants may only refer to immutable values
+    let reborrow = {p};
```

Comment:
This is an improvement on the old code, but, just out of curiosity, do you know what caused this to change?

Reply:
Reborrows are now special cased in `HasMutInterior::in_rvalue`.